app/src/main/res/layout/fragment_all_characters.xml

```diff
@@ -22,22 +22,10 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"/>
 
-    <TextView
-        android:id="@+id/fragment_all_characters_character_id"
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/fragment_all_characters_recycler_view"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:textSize="@dimen/text_size_character_id"
-        android:textColor="@android:color/white"
-        android:textAlignment="center"
-        app:layout_constraintTop_toTopOf="parent"/>
-
-    <TextView
-        android:id="@+id/fragment_all_characters_count"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:textAlignment="center"
-        android:textColor="@android:color/white"
-        android:textSize="@dimen/text_size_character_id"
-        app:layout_constraintTop_toBottomOf="@id/fragment_all_characters_character_id"/>
+        android:layout_height="match_parent"
+        android:visibility="gone"/>
 
```

Comment:
tip: you can add:
`tools:visibility="gone"` 
` tools:listitem="@layout/item_timeline"/>` to see on your editor, a preview of how your recycle looks.

Reply:
Yes, I did, but how I need the visibility of the recycler gone by default, is the same because do not show anything. Thanks anyway Lucho!